libs/langchain/langchain/document_loaders/recursive_url_loader.py

```diff
@@ -1,11 +1,30 @@
-from typing import Iterator, List, Optional, Set
+import re
+from typing import Callable, Iterator, List, Optional, Set, Union
 from urllib.parse import urljoin, urlparse
 
 import requests
 
 from langchain.docstore.document import Document
 from langchain.document_loaders.base import BaseLoader
 
+try:
+    import aiohttp
+except ImportError:
+    print("The aiohttp package is required for the RecursiveUrlLoader.")
+    print("Please install it with `pip install aiohttp`.")
+
+try:
+    import asyncio
```

Comment:
This is python  stdlib so no need to try/catch
https://docs.python.org/3/library/asyncio.html